exercises/binary/binary_test.rb

```diff
@@ -66,6 +66,6 @@ def test_invalid_binary_numbers_raise_an_error
   # http://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/constants.html
   def test_bookkeeping
     skip
-    assert_equal 1, Binary::VERSION
+    assert_equal 2, Binary::VERSION
```

Comment:
Looks good to me.  

The version bump would be required, if it can cause currently passing submissions to fail.